JS_HW/task_2.js

```diff
@@ -0,0 +1,15 @@
+//   У прямоугольного треугольника две стороны n (со значением 3) и m (со значением 4).
+//  Найдите гипотенузу k по теореме.
+
+
+
+
+function k (n, m) {
+
+    console.log(Math.sqrt(Math.pow(n, 2) + Math.pow(m, 2)));
+
+    return;
```

Comment:
see the comment above. Make sure that all of your functions return the result, not just console.log